MaxText/accelerator_to_spec_map.py

```diff
@@ -45,6 +45,9 @@ class SystemCharacteristics:
     "v6e-128": SystemCharacteristics("tpu", "v6e:8x16", "default", (2, 2, 1), 128, (False, True, False)),
     "v6e-256": SystemCharacteristics("tpu", "v6e:16x16", "default", (2, 2, 1), 256, (True, True, False)),
     # v5e: one core per chip with 16 GB HBM
+    "v5e-1": SystemCharacteristics("tpu", "v5e:1x1", "default", (2, 2, 1), 1, (False, False, False)),
```

Comment:
v5e-1 has one chip so does it use the same (2,2,1) per host/vm chips_per_host_bounds ? Please see v6e-1 above.

Reply:
Great catch!